docs/io.cozy.bills.md

```diff
@@ -16,6 +16,15 @@
 
 ### Optional attributes (but some are important depending the context)
 
+- `recurring`: {String} : is the bill a recurring one ?
+    - `weekly`: the bill is recurring every week in the day of the week corresponding to the `date` attribute.
+    - `monthly`: the bill is recurring every month in the day corresponding to the `date` attribute.
+    - `bi-monthly`: the bill is recurring every 2 months in the day corresponding to the `date` attribute.
+    - `yearly`: the bill is recurring every year in the day of the year corresponding to the `date` attribute (be careful of leap year).
+      Ex: date: 28/12/2019, recurring: monthly -> cron expression 0 0 0 28 * *
+          date: 31/12/2019, recurring: monthly -> cron expression 0 0 0 28-31 * * (but the service
+          will need to find a clever way to have only one recurring operation by month.
```

Comment:
I don't understand the 28-31 bit in the second example ?

Reply:
A cron expression does not handle last months properly. You don't want a monthly recurring bill every 31th of the month to happen only every 2 month because not every month has a 31.

The doctype will not handle this. But the service using this doctype will to handle this properly.

Reply:
I think it is a bit prematured at thie point (just below yearly) to have cron expressions etc.. It's a bit surprising here, I think it would be more suited in a "How does a user of recurrence should find associated operations" section ? But still, I am not sure it belongs here.

Reply:
Alright, I think I will remove the cron expression thing, then